h2o-core/src/main/java/hex/grid/Grid.java

```diff
@@ -538,6 +540,24 @@ public void exportModelsBinary(final String exportDir) throws IOException {
     }
   }
 
+  /**
+   * Saves all of the frames used by this Grid's params. Frames are named by their keys.
+   * @param exportDir directory to export all the frames to
+   */
+  public void exportFramesBinary(final String exportDir) {
+    Map<String, Frame> frames = getParams().getAllFrames();
+    frames.forEach((name, f) -> new FramePersist(f).saveTo(exportDir, true));
```

Comment:
- The `saveTo` operation can potentially fail on a subset of persisted frames. There are runtimeExceptions thrown, which would cause the underlying Job to fail.
- `name` unused
- It is unclear whether this method blocks until all the frames are exported or not. This brings up yet another question - should be block on this operation (`exportFramesBinary`) and verify the output before moving on to next round ?

A rough proposal that both blocks and checks for errors (but only logs the errors, does not act upon them in any way):
```suggestion
    final List<Job<Frame>> crashedFrames = frames.values().stream()
            .map((f) -> new FramePersist(f).saveTo(exportDir, true))
            .filter(job -> {
              job.get(); // Wait for the persist operation ot complete.
              return job.isCrashed();
            })
            .collect(Collectors.toList());
    if(!crashedFrames.isEmpty()){
      crashedFrames.forEach(frame -> {
        Log.err(String.format("Unable to export binary frame '%s'", frame.get()._key.toString()));
      });
    }
```